dist/test_data.py

```diff
@@ -132,7 +132,9 @@ def __ge__(self, other):
            operation.''',
         min=0, max=1),
     Config('free_space_target_mb', '20', r'''
-        Minimum amount of space in MB recoverable for compaction to proceed.''')
+        Minimum amount of space in MB recoverable for compaction to proceed.'''),
+    Config('op_rate', '60s', r'''
```

Comment:
We could replace this with the existing `throttle_config` and define `60s` in the config.

Reply:
I see, so would that look like:
```suggestion
    background_compact_thread_config = throttle_config + [
        Config('thread_count', 0, r'''
        Specifies the number of threads that will be used to perform background compaction
           operation.''',
        min=0, max=1),
        Config('free_space_target_mb', '20', r'''
        Minimum amount of space in MB recoverable for compaction to proceed.'''),
    ]
```

Reply:
yes!

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/pull/9691/commits/b21630dc0d16a3e94d7d729cc9d519ab60c84c7e.